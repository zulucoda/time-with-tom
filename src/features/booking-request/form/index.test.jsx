import { render } from '@testing-library/react';
import BookingRequestForm from '.';
import React from 'react';

describe('Features - Booking Request - Form - Unit Test', function () {
  test('does it render', () => {
    const { container } = render(<BookingRequestForm />);
    expect(container).toMatchSnapshot();
  });
});
