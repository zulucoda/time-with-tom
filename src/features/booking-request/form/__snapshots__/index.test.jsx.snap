// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features - Booking Request - Form - Unit Test does it render 1`] = `
<div>
  <form
    action="#"
  >
    <div>
      <select
        name="activity"
      >
        <option
          value="none"
        >
          Please select
        </option>
        <option
          value="golf"
        >
          Golf
        </option>
        <option
          value="tennis"
        >
          Tennis
        </option>
        <option
          value="squash"
        >
          Squash
        </option>
        <option
          value="trail_run"
        >
          Trail run
        </option>
        <option
          value="cycling"
        >
          Cycling
        </option>
      </select>
    </div>
    <pre>
      debug value
      {
   "activity": "none"
}
    </pre>
    <pre>
      debug errors
      {}
    </pre>
    <button
      type="submit"
    >
       Submit 
    </button>
  </form>
</div>
`;
